showcase/app/templates/components/app-header/index.hbs

```diff
@@ -337,51 +421,67 @@
 
   <Shw::Text::H2>Base elements</Shw::Text::H2>
 
-  <Shw::Text::H3>SideNav::Header::HomeLink</Shw::Text::H3>
+  <Shw::Text::H3>AppHeader::HomeLink</Shw::Text::H3>
 
   <Shw::Text::H4>Content</Shw::Text::H4>
 
   <Shw::Flex as |SF|>
     <SF.Item @label="Icon">
       <div class="hds-app-header">
-        <Hds::AppHeader::HomeLink @icon="hashicorp" @ariaLabel="HashiCorp" @href="#" />
+        <Hds::AppHeader::HomeLink @icon="hashicorp" @text="HashiCorp" @href="#" />
       </div>
     </SF.Item>
 
     <SF.Item @label="Custom color">
       <div class="hds-app-header">
         <Hds::AppHeader::HomeLink
           @icon="boundary"
-          @ariaLabel="Boundary"
+          @text="Boundary"
           @color="var(--token-color-boundary-brand)"
           @href="#"
         />
       </div>
     </SF.Item>
   </Shw::Flex>
 
+  <Shw::Text::H4>Text</Shw::Text::H4>
+
+  <Shw::Flex as |SF|>
+    <SF.Item>
+      <div class="hds-app-header">
+        <Hds::AppHeader::HomeLink @icon="hashicorp" @text="HashiCorp" @href="#" @title="Admin UI" />
+      </div>
+    </SF.Item>
+
+    <SF.Item>
+      <div class="hds-app-header">
+        <Hds::AppHeader::HomeLink @icon="terraform" @text="Terraform" @href="#" @title="Terraform Admin Console" />
```

Comment:
[todo] I think this one is missing the `@isIconOnly={{false}}`?


```suggestion
        <Hds::AppHeader::HomeLink @icon="terraform" @isIconOnly={{false}} @text="Terraform" @href="#" @title="Terraform Admin Console" />
```

Reply:
Shoot, I did update this, but I did on my docs branch. Let me reconcile those.